src/lib.rs

```diff
@@ -130,6 +130,170 @@ impl<'a> Drop for Sentinel<'a> {
     }
 }
 
+/// `ThreadPool` factory, which can be used in order to configure the properties of the
+/// `ThreadPool`.
+///
+/// The three configuration options available:
+///
+/// * `num_threads`: maximum number of threads that will be alive at any given moment by the built
+///   `ThreadPool`
```

Comment:
It would be nice if this and other references to `ThreadPool` were made into links to `struct.ThreadPool.html`, so that the [C-LINK](https://rust-lang-nursery.github.io/api-guidelines/documentation.html#prose-contains-hyperlinks-to-relevant-things-c-link) guideline was met. The existing documentation is updated in #85.